data/README.md

```diff
@@ -351,3 +353,37 @@ pq.write_table(
 
 This is a practical case where `BYTE_STREAM_SPLIT` encoding obtains a smaller file size than `PLAIN` or dictionary.
 Since the distributions are random normals centered at 0, each byte has nontrivial behavior.
+
+# Additional types
+
+`byte_stream_split_extended.gzip.parquet` is generated by pyarrow 16.0.0.
+It contains 7 pairs of columns, each in two variants containing the same
+values: one `PLAIN`-encoded and one `BYTE_STREAM_SPLIT`-encoded:
+```
+Version: 2.6
```

Comment:
hmm... it seems we need to fix the version so here can correctly displays version 2.11

Reply:
That will have to be part of the final C++ PR. We do not need the C++ changes to fix this README if we want to, though.

Reply:
(also see https://github.com/apache/arrow/issues/40096)